test/integration/README.md

```diff
@@ -0,0 +1,29 @@
+# Integration tests
+
+Integration tests exercise the OpenShift Ansible playbooks by running them
+against an inventory with Docker containers as hosts.
+
+## Requirements
+
+The tests assume that:
+
+* docker is running on localhost and the present user has access to use it.
+* golang is installed and the go binary is in PATH.
+
+## Running the tests
+
+From the repository root, run with:
+
+```
+tox -e integration
+```
+
+You can also run them more directly, for example to run specific checks
+or increase concurrency:
+
+```
+go test ./test/integration/... -p 8 -run TestPackageUpdateDepMissing
```

Comment:
did not know that, interesting. Actually the test script will provide this option, so I'll just leave it out of the README.